pkg/client/proxy_snapshot.go

```diff
@@ -120,13 +120,14 @@ func (c *ProxyClient) SnapshotList(backendStoreDriver, engineName, volumeName,
 }
 
 func (c *ProxyClient) SnapshotClone(backendStoreDriver, engineName, volumeName, serviceAddress, snapshotName,
-	fromController string, fileSyncHTTPClientTimeout int) (err error) {
+	fromControllerAddress, fromControllerName string, fileSyncHTTPClientTimeout int) (err error) {
 	input := map[string]string{
-		"engineName":     engineName,
-		"volumeName":     volumeName,
-		"serviceAddress": serviceAddress,
-		"snapshotName":   snapshotName,
-		"fromController": fromController,
+		"engineName":            engineName,
+		"volumeName":            volumeName,
+		"serviceAddress":        serviceAddress,
+		"snapshotName":          snapshotName,
+		"fromControllerAddress": fromControllerAddress,
+		"fromControllerName":    fromControllerName,
```

Comment:
NIT: `fromControllerName` vs `engineName` looks weird... Can you help correct this part with uniform naming?